system/languages/en.php

```diff
@@ -358,6 +358,7 @@
 
 		// help_icon_texts
 		"help-private-page" => "A private page is only visible to users who are logged in.",
+		"help-page-type" => "choose your pagetype first. \nA festival page will give you the tools to organise a festival and show the line-up to the visitor. \nA web page is a regular weg page. \nA mailinglist creates a list for emailaddresses and a tool to send mailings. \nA peer reviewed article creates a page that remains private until 3 people have approved it, \nthen it becomes public with a comments possibility. \nAweb page creates a regular web page.",
```

Comment:
I think each line of this helptext should start with a capital letter? And I think there is no point in having spaces before each `\n`?

I also don't really like hardcoding this list of possible page types, since we (or users) might add new datatypes which are then not reflected here. However, to properly solve this, datatypes must be able to provide their own helptext fragment somehow. But that is complicated, so until we have that, this is probably a fine approach (but we should create an issue to track this once this is merged).